src/openklant/tests/vcr.py

```diff
@@ -1,7 +1,7 @@
 import os
 from pathlib import Path
 
-from vcr.unittest import VCRMixin as _VCRMixin
+from maykin_common.vcr import VCRMixin as _VCRMixin
```

Comment:
you can simplify the derived mixin now and remove the requirement to define `VCR_TEST_FILES` - it builds a sane directory by default: https://maykin-django-common.readthedocs.io/en/latest/reference/vcr.html#maykin_common.vcr.VCRMixin